### PR TITLE
Update history page

### PR DESCRIPTION
**Pull request by RobinKnipe**
Some minor content changes:
image

### CODE REVIEW

**Comment by daniel-ac-martin** [comment]
The ticks are a bit confusing as they seem to correspond to the months rather than the quarters. Is that new?

File: src/lev-report/charts/CumulativeUsage.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const CumulativeUsage = ({ data }) => data && data.length &&
 			tickFormat={tick => {
 				const date = moment.unix(tick);
 				const month = date.month();
-				return month ? month % 3 === 0 ? `Q${date.quarter()}` : '' : `'${date.format('YY')}`;
+				return month ? month % 3 === 0 ? `Q${date.quarter()}` : '' : date.format('YYYY');
```
Comment (daniel-ac-martin) [inline src/lev-report/charts/CumulativeUsage.js:22]: Would be nice to have some parens here!

I know the lack of parens wasn't caused by this PR though so it's up to you.

Comment (daniel-ac-martin) [reply src/lev-report/charts/CumulativeUsage.js:22]: On a more relevant subject, why move away from `'20`? - I'd have thought that's more concise and easy to read.

Comment (daniel-ac-martin) [reply src/lev-report/charts/CumulativeUsage.js:22]: Or maybe we should add the full year but remove the quarters...

Comment (RobinKnipe) [reply src/lev-report/charts/CumulativeUsage.js]: Brackets added for clarity.
Full years was an old request that came from Nigel.

**Comment by RobinKnipe** [comment]
Plus some improvements to the tick and grid lines, to make reading the chart easier:
image

File: src/lev-report/charts/CumulativeUsage.js (hunk 1 above)
Comment (daniel-ac-martin) [reply src/lev-report/charts/CumulativeUsage.js]: Okay.

It will be nice when we've got Chromatic in place and we can do proper visual reviews of things.

**Comment by daniel-ac-martin** [comment]
That's quite nice.

**Comment by RobinKnipe** [comment]
> The ticks are a bit confusing as they seem to correspond to the months rather than the quarters. Is that new?

Ah, somehow I missed this, but hopefully it's been addressed by the latest changes?

**Comment by daniel-ac-martin** [comment]
Yeah it definitely helps.

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**